dt/uni04delta-ipv6/kustomization.yaml

```diff
@@ -194,15 +194,3 @@ replacements:
           - spec.nova.template.cellTemplates
         options:
           create: true
-
-  - source:
```

Comment:
we should keep this

Reply:
Imo we should not – in other uni jobs we do not need to specify that explicitly, it gets the default from lib/control-plane/openstackcontrolplane.yaml